web/js/components/draggable.js

```diff
@@ -77,8 +78,9 @@
         case touchend:
         case 'contextmenu':
           this.clearTimer();
-
           break;
+        default:
+          throw new Error(`Unknown event ${evt.type}`);
```

Comment:
I see you are adding `default`s to many `switch`es throughout the codebase. I'm wondering if this might break anything (since previously, there was no `default` options, resulting in a no-op). Would it be better to do a no-op (rather than throw an error):

```
default:
  // No-op on default;
```

If you think throwing an error is OK, that's fine. Just wondering.

Reply:
I think its ok to throw the error because i think reallistically we will never hit the default case. On the other hand, i do agree that its safe to no-op. will make the change